lib/Inkscape.js

```diff
@@ -1,9 +1,21 @@
 const childProcess = require('child_process');
+const semver = require('semver');
 const Stream = require('stream').Stream;
 const fs = require('fs');
 const getTemporaryFilePath = require('gettemporaryfilepath');
 const async = require('async');
 
+// check if the installed inkscape version is compatible
+const versionChild = childProcess.spawnSync('inkscape', ['--version']);
```

Comment:
This makes the module 150-200ms slower to require on my machine, so it would be nice to avoid it. OTOH I can't really offer a good alternative until inkscape 1.x is more widespread.

Since impro is bring-your-own-engines it's probably fine.